Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
We use OpenJDK in production without any problem.
I would go with OpenJDK... and if we encounter problems, we'll switch to Oracle.